packages/cli/src/utils/getCommandNameFromArgs.ts

```diff
@@ -0,0 +1,9 @@
+import { Arguments } from 'yargs';
+
+export function getCommandNameFromArgs(argv: Arguments | undefined): string | number {
+  if (!argv || !argv._ || !argv._.length) {
+    return '';
+  }
+
+  return argv._[0];
```

Comment:
Would this also work? 

```suggestion
  return argv?._?.[0] ?? '';
```

Reply:
Can be shorter, applied.